Boyardee-B update

ToasterBan opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Updates the Boyardee-B, list of all changes will be done later

the main intent of these changes is to make the boyardee-B a more combat capable but still restaurant capable ship

- removed the vault, replaced with a botany/coldroom area
- the carp and punpuns have been removed, replaced by Bitey the alligator
- added the syndicate waiter suit and respective job which is the assistant role of the Boyardee-B
- AAC starts unlocked
- minor tile changes so carpets look a bit better
- Suit Storage's are now equipped with syndie masks and suits (not hardsuits)
- Added some extra engines and more fuel for said engines allowing it to be better for combat
- regular pacmans replaced with superpacmans
- added a 3 use emag, use it wisely
- added dice and a roulette machine to add to the casino aesthetic 
- fixed some small lighting issues
- 

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
the normal version got an update so why not give its edgier brother a little love?
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Boyardee-B class updates
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

ToasterBan committed:
update map

ToasterBan committed:
config update

ToasterBan committed:
syndie suit

ToasterBan committed:
small fixes

ToasterBan committed:
Merge remote-tracking branch 'upstream/master' into Boyardee-Patch-2

ToasterBan committed:
jobs and fixes

Calyxman commented:
Pretty cool, what about adding a Clown job to it? Keep the safe, put it somewhere under the floor though.

ToasterBan commented:
a clown role could be interesting but the main idea if the ship is that it's a restaurant-casino and I don't know how clowns will fit in

Calyxman commented:
> a clown role could be interesting but the main idea if the ship is that it's a restaurant-casino and I don't know how clowns will fit in

the clown encourages you to gamble.
in all seriousness i feel like not too many players gamble, let alone interact with boyardees frequently. it makes sense for an entertainment vessel to have entertainers (although the clown would be a little hard to make space for)
also holy shit we need this speedmerged

Calyxman commented:
finally tmerged. is the exclusion of a nutrient vendor/botany tools intentional?

ToasterBan commented:
> finally tmerged. is the exclusion of a nutrient vendor/botany tools intentional?

yes, you are trading some restaurant-ness for weapons and a faster ship

ToasterBan commented on _maps/shuttles/shiptest/bar_ship_b.dmm:
```diff
@@ -2308,28 +2323,40 @@
+/mob/living/simple_animal/hostile/retaliate/gator/steppy{
+	attack_verb_continuous = "tries to bite";
+	attack_verb_simple = "tries to bite";
+	desc = "Bitey the gator! it appears to have lost its teeth";
```
```suggestion
	desc = "Bitey the gator! It appears to have lost its teeth";
```
spelloing

ToasterBan committed:
fixes spelling error

ToasterBan committed:
fixes an engine + minor touchups

Calyxman commented:
> > finally tmerged. is the exclusion of a nutrient vendor/botany tools intentional?
> 
> yes, you are trading some restaurant-ness for weapons and a faster ship

a little harder to do botany without, like, a cultivator. iirc you can make pest spray in the biogen and nutrients in the biogen as well, but you can't make cultivators.

ToasterBan committed:
botany update

ToasterBan commented:
alright added some botany tools, botany should be doable while the "restaurant is weaker" element still taking part

Calyxman commented:
> alright added some botany tools, botany should be doable while the "restaurant is weaker" element still taking part

Based ship dev?! Listens to criticisms? Thank you man

ToasterBan commented:
> > alright added some botany tools, botany should be doable while the "restaurant is weaker" element still taking part
> 
> Based ship dev?! Listens to criticisms? Thank you man

The entire point of this pr is to improve the ship for everyone

I am planning on updating other ships but I am less accustomed to them so I highly reccomend going on the forums and throwing your suggestions in the ship's thread in here: https://forums.shiptest.net/c/coder-hell/map-review/15

ToasterBan committed:
more pacmen

the boyardee-b lacks many QOL items and items deemed necessary by some so an extra pacman as a backup due to uranium being rare won't be that much of an issue
